test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperTextAreaExtensionsTest.cs

```diff
@@ -140,7 +140,7 @@ public void TextAreaFor_UsesSpecifiedRowsAndColumns()
 
             // Assert
             Assert.Equal(
-                "<textarea attr=\"HtmlEncode[[value]]\" columns=\"HtmlEncode[[2]]\" id=\"HtmlEncode[[Property1]]\" " +
+                "<textarea attr=\"HtmlEncode[[value]]\" cols=\"HtmlEncode[[2]]\" id=\"HtmlEncode[[Property1]]\" " +
```

Comment:
@ajaybhargavb looks like we only have 2 tests using the `TextArea[For]()` `columns` argument. Please have a quick look at the old Web Stack Runtime tests to see if we're missing any important cases.